test/unit/domain/wallets/tx-history.spec.ts

```diff
@@ -33,15 +35,21 @@ describe("translates ledger txs to wallet txs", () => {
   const ledgerTxnsInputs = ({
     walletId,
     settlementAmount,
-    usd,
+    centsAmount,
```

Comment:
shouldn't this add satsAmount?

Reply:
https://github.com/GaloyMoney/galoy/pull/2191#discussion_r1084196607